api/util/mappers/engine.py

```diff
@@ -333,7 +321,6 @@ def map_environment_to_engine(
         #
         # Integrations:
         amplitude_config=amplitude_config_model,
-        dynatrace_config=dynatrace_config_model,
```

Comment:
I'm unsure why this has been removed here?

Reply:
Dynatrace integration isn't currently (or wasn't ever) an identity one. We don't need it indexed for Edge API.